src/cbc_sdk/workload/vm_workloads_search.py

```diff
@@ -55,9 +56,32 @@ def _query_implementation(cls, cb, **kwargs):
             **kwargs (dict): Not used, retained for compatibility.
 
         Returns:
-            ComputeResourceQuery: The query object
+            VCenterComputeResourceQuery: The query object
```

Comment:
If this is just raising an exception, do we need the return section?

Reply:
It keeps the documentation consistent.